ambari-admin/src/main/resources/ui/admin-web/bower.json

```diff
@@ -3,21 +3,24 @@
   "private": true,
   "dependencies": {
     "bootstrap": "3.3.7",
-    "angular": "1.7.5",
-    "angular-route": "1.7.5",
+    "angular": "*",
+    "angular-route": "*",
```

Comment:
This gets latest version only right?

Reply:
To be honest, I'm not familiar with angular, but I think it'll be better we keep less change from release 2.7.6, which in this case, is ported from release 2.7.6

Reply:
Sounds good. +1 (non-binding) from my side. Thanks @kevinw66